libs/nx-vite/README.md

```diff
@@ -0,0 +1,41 @@
+# nx-vite
+
+NX Plugin for Next Generation Frontend Tooling with Vite
+
+## Generators
+
+### node
```

Comment:
Need to update to match what has been done

Reply:
Updated :)